test/functional/terminal/edit_spec.lua

```diff
@@ -36,6 +36,9 @@ describe(':edit term://*', function()
     local scr = get_screen(columns, lines)
     local rep = 'a'
     meths.set_option('shellcmdflag', 'REP ' .. rep)
+    if helpers.os_name() == 'windows' then
+      command('set shellxquote=')
```

Comment:
This is also the default for unix, so just do it unconditionally.

Reply:
I prefer `outer quotes` or something to describe `'"' .. command .. '"'` for expected output.